movie/src/main/res/layout/activity_home.xml

```diff
@@ -14,4 +14,18 @@
         app:defaultNavHost="true"
         app:navGraph="@navigation/nav_home" />
 
+    <TextView
+        android:id="@+id/no_internet_warning"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_gravity="bottom"
+        android:background="@color/colorPrimary"
+        android:gravity="center"
+        android:text="@string/internet_unavailable"
+        android:textAllCaps="true"
+        android:textColor="@android:color/holo_red_light"
```

Comment:
Nit: I don't like to rely on Android's color palette. Could you please create a color in `colors.xml` and then reference Android's if you like :)